### ISSUE TITLE
Problem with cweagans/composer-patches when running update-dependencies job

### ISSUE THREAD

**Issue by m4olivei** [deviantintegral/drupal_tests#31]
After setting up a new repo with the latest master script, and running the jobs on CircleCI, I got this error from composer:

```
[Composer\DependencyResolver\SolverProblemsException]                                                                                                  
  Problem 1                                                                                                                                              
      - The requested package cweagans/composer-patches ~1.0 exists as cweagans/composer-patches[dev-master] but these are rejected by your constraint.
```

It seems like the version constraint for `cweagans/composer-patches` in the root composer.json (which is `~1.0`, located at `/var/www/html/composer.json`) and the version constraint for `cweagans/composer-patches` in the module set by the script (`dev-master`) do not agree, and that causes an issue.

**Comment by m4olivei** [comment]
Changing the version constraint in my modules `composer.json` to `cweagans/composer-patches:~1.0` fixed it for me.

**Comment by m4olivei** [comment]
I re-ran though this using a public repo and the same issue happened. Here is the CircleCI job that failed:

https://circleci.com/gh/m4olivei/drupal_ci_test/1

**Comment by deviantintegral** [comment]
Fixed in 5505faf
